src/Mod/Path/PathScripts/PathPocketShape.py

```diff
@@ -389,19 +389,19 @@ def areaOpShapes(self, obj):
             self.outlines = [Part.Face(TechDraw.findShapeOutline(base.Shape, 1, FreeCAD.Vector(0, 0, 1))) for base in self.model]
             stockBB = self.stock.Shape.BoundBox
 
-            self.removalshapes = []
             self.bodies = []
             for outline in self.outlines:
                 outline.translate(FreeCAD.Vector(0, 0, stockBB.ZMin - 1))
                 body = outline.extrude(FreeCAD.Vector(0, 0, stockBB.ZLength + 2))
                 self.bodies.append(body)
                 self.removalshapes.append((self.stock.Shape.cut(body), False))
 
-        for (shape, hole) in self.removalshapes:
+        for (shape, __) in self.removalshapes:
```

Comment:
What's going on here?

Reply:
Not sure about what you are asking.
* I learned from Schildkroet that I can place a single or double underscore as place holders for unused tuple items when unpacking to appease LGTM checks. 
* The `self.removalshapes` variable is a list of tuples to be passed to PathAreaOp for processing.  The tuples are (envelope shape, is hole boolean) pairs.
* The existing `obj.removalshape` is only storing the first shape in the `self.removalshapes` list.  This new code combines all the envelope shapes into a `Part.Compound` shape for reference to the collective removal shape passed to PathAreaOp.

Do any of these responses address your question?

Reply:
Yes.  I've never seen that trick. If we do unusual things to appease LGTM or travis or other tools and it isn't a python syntax trick, please add a comment.  Otherwise it looks like a mistake.